packages/cli/test/integration-1.test.ts

```diff
@@ -521,8 +522,11 @@ test('deploy using --local-config flag above target', async () => {
 });
 
 // eslint-disable-next-line jest/no-disabled-tests
-test.skip('deploy `api-env` fixture and test `vercel env` command', async () => {
+test('deploy `api-env` fixture and test `vercel env` command', async () => {
   const target = await setupE2EFixture('api-env');
+  const promptEnvVar = `VAR_${randomBytes(8).toString('hex')}`;
```

Comment:
would `chance` be a better fit here?

Reply:
I strongly discourage use of randomness in standard test suites. It makes the tests brittle without providing value, in my opinion. Instead, I would pick some arbitrary fixed value.

Reply:
I (generally) strongly discourage use of randomness in standard test suites. It makes the tests brittle without providing value, in my opinion.

Here, the goal is to make sure these values are unique among other runs, which is a unique situation. Randomness can be fine for this purpose, but I would add a comment about why this is necessary. Otherwise, I might forget in the future and come by to remove this randomness.